### PR TITLE
Add proto stub for message to get usage data

### PR DESCRIPTION
**Pull request by john-cloudnatix**

### CODE REVIEW

File: api/v1/usage_server.proto [hunk 1]
```diff
@@ -28,6 +28,56 @@ message AggregatedSummary {
   repeated Summary method_summaries = 2;
 }
 
+message GetUsageDataRequest {
+  string tenant_id = 1;
```
Comment (kkaneda) [inline api/v1/usage_server.proto:32]: Let's remove this. Once we provide the externally visible endpoint, tenant_id will be hidden from end users.

File: api/v1/usage_server.proto [hunk 2]
```diff
@@ -28,6 +28,56 @@ message AggregatedSummary {
...
+  message Filter {
+    repeated string api_keys = 1;
+    repeated string models = 2;
+  }
+}
+
+
+
+message UsageData {
+  string method = 1;
```
Comment (kkaneda) [inline api/v1/usage_server.proto:57]: Could you explain what this field means?

File: api/v1/usage_server.proto [hunk 3]
```diff
@@ -28,6 +28,56 @@ message AggregatedSummary {
...
+  repeated UsageDataPoint data_points = 2;
+}
+
+message UsageDataPoint {
+  string user_id = 1;
+  string tenant = 2;
```
Comment (kkaneda) [inline api/v1/usage_server.proto:63]: could you remove this as well?

**Review by kkaneda — APPROVED** [review]
Have some minor comments, but looks good!

File: api/v1/usage_server.proto (hunk 2 above)
Comment (john-cloudnatix) [reply api/v1/usage_server.proto]: removed it, I think it snuck in with co-pilot

**Pull request merged**